README.md

```diff
@@ -136,13 +139,17 @@ allow-licenses:
   - 'GPL-3.0'
   - 'BSD-3-Clause'
   - 'MIT'
+allow-dependencies-licenses:
+  npm: @actions/core, @actions/github
+  pip: requests
```

Comment:
I see what you mean. I added it to the example here for a similar reason as my previous comment in the options table... Since it was mentioning `allow-licenses`, I though it would be a good spot to exemplify the options to exclude packages. I will remove it for now as per your suggestion.

Would creating a `docs/examples.md` or `docs/advanced-usage.md` be an option? We could link to it from this README to more configuration examples and possible uses of this action?

Reply:
> We could link to it from this README to more configuration examples and possible uses of this action?

I love this idea, it'd allow us to show multiple examples of more advanced usage.

Reply:
I've added a file in [1896d6f](https://github.com/actions/dependency-review-action/pull/423/commits/1896d6f93643a7d934aa99e1a335e8e5a3199e27) and put in first few examples that came to mind. 

I guess we will have to restructure as hopefully the examples grow but I think this works for now.